### ISSUE TITLE
Apple Pay Button has no Alt Text/Description for Screenreader Users. just says 'Button'

### ISSUE THREAD

**Issue by TimNoonanVoice** [woocommerce/woocommerce-gateway-stripe#227]
Site visitors using screenreaders do not hear a reference to Apple Pay when they navigate to the Pay with Apple Pay button. Alll they hear is 'Button'.
Pay with Paypal and other options speak correctly.  

#### Steps to reproduce the issue

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

**Comment by dougaitken** [comment]
@roykho are we able to revisit this, please? Chatting with @TimNoonanVoice on Twitter:

"Instead the "Buy with ApplePay" button needs to be markedup with aria-label - [details](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)"

[Twitter thread](https://twitter.com/TimNoonan/status/925148194769485824)
